tests/startup_capture_test.go

```diff
@@ -41,10 +40,7 @@ func TestStartupCapture(t *testing.T) {
 				return err
 			}
 
-			gremlin := config.GetString("analyzer.startup.capture_gremlin")
-			if gremlin == "" {
-				return fmt.Errorf("No startup capture is defined in config!")
-			}
+			gremlin := "g.V().Has('Name','startup-vm2')"
```

Comment:
nit: space between parameters

Reply:
nop, except if you can generate a yaml file with ansible that contain a space

Reply:
in a thu a Vagrant file variable ...
contrib/vagrant/Vagrantfile:  $skydive_extra_config["analyzer.startup.capture_gremlin"] = "g.V().Has('Name','startup-vm2')"

challange accepted ? :)